homeassistant/components/netatmo/climate.py

```diff
@@ -118,6 +118,8 @@ async def async_setup_entry(
 
     @callback
     def _create_entity(netatmo_device: NetatmoRoom) -> None:
+        if not netatmo_device.room.climate_type:
+            raise ValueError("No climate type found for this room")
```

Comment:
Why still raise an error? Why not invert this if statement and only add when we have one, otherwise we log?

Reply:
Thanks for pushing me. You are right.